app/Libraries/Payments/PaypalExecutePayment.php

```diff
@@ -63,7 +63,7 @@ public function run()
 
                 return $payment->execute($this->execution, $context);
             } catch (PayPalConnectionException $e) {
-                \Log::error($e->getData());
+                \Log::error($e->getMessage());
```

Comment:
I don't think `Log::error` sends sentry log...? :thinking:

Reply:
sentry-laravel add handlers to logging events and adds breadcrumbs when a message is logged